tests/test_quantum_info_metrics.py

```diff
@@ -311,6 +311,7 @@ def test_process_fidelity_and_infidelity(backend):
     )
 
 
+@pytest.mark.skip
```

Comment:
I'd open an issue linked to https://github.com/qiboteam/qibo/pull/1231#issuecomment-1997932717 to remember that this may be possible to fix at some point.

Reply:
Feel free :)

Reply:
#1298 1298